detekt-rules/src/main/kotlin/OrtPackageNaming.kt

```diff
@@ -48,7 +48,7 @@ class OrtPackageNaming : Rule() {
     override fun visitPackageDirective(directive: KtPackageDirective) {
         super.visitPackageDirective(directive)
 
-        if (directive.qualifiedName.isEmpty()) return
+        if (directive.qualifiedName.isEmpty() || directive.qualifiedName.startsWith("test.")) return
```

Comment:
Please add a code comment that this change is *only* required for `OrtLogTestExtension`.